src/plugin/storage/persistplugin.js

```diff
@@ -97,13 +100,15 @@ class PersistPlugin extends AbstractPlugin {
         }
       });
     } else {
+      let msg = '';
       if (window.isSecureContext === false) {
-        msgs.push(`The application is not being served in a secure context. Persistent storage permission
-            will be automatically declined.`);
+        msg = `This application is not being served in a secure context. The persistent storage permission
+            will be automatically declined. `;
       }
 
-      msgs.push('Your application settings may be automatically reset when under storage pressure.');
-      this.log(msgs);
+      msg += `Your application settings may be automatically reset when under storage pressure. Get more info 
+        <a href="` + helpUrl + `"rel="noopener" target="_blank">here</a>`;
```

Comment:
String concatenation isn't necessary for [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). It's a bit more readable to use it like so:
```javascript
msg += `Your application settings may be automatically reset when under storage pressure. Get more info 
        <a href="${helpUrl}" rel="noopener" target="_blank">here</a>`;
```
If OpenSphere doesn't care about it, then either way is okay. Just a bit strange to look at to me.
Same for the other instances in the changes.

Reply:
Agree, if you're using template literals you should use expressions instead of string concatenation.

Reply:
I made that change